libkbfs/journal_server_util.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import "errors"
+
+// GetJournalServer returns the JournalServer tied to a particular
+// config.
+func GetJournalServer(config Config) (*JournalServer, error) {
```

Comment:
Hmm...it can, but then all the implementations would be the same, so a function like this would still be needed, I think.

Are you suggesting having a `JournalServer` field in `Config` like everything else?